dgbpy/mlapply.py

```diff
@@ -376,9 +376,13 @@ def doTrain( examplefilenm, platform=dgbkeys.kerasplfnm, type=TrainType.New,
                                           nbchunks=params['nbchunk'],
                                           force=False,
                                           split=params['split'],nbfolds=params['nbfold'] )
-      if type != TrainType.New and dgbkeys.trainconfigdictstr in infos:
-        params[dgbkeys.criteriondictstr] = infos[dgbkeys.trainconfigdictstr][dgbkeys.criteriondictstr]
+      if type != TrainType.New and dgbkeys.trainconfigdictstr in infos and dgbkeys.withunlabeleddictstr in dgbmlio.getInfo(examplefilenm):
```

Comment:
use dgbpy.hdf5.hasUnlabeled( infos )
Store it as a local variable, so you do not need to repeat that function call